mautrix_facebook/portal.py

```diff
@@ -2012,6 +2029,30 @@ async def handle_facebook_reaction_remove(
                 pass
             await reaction.delete()
 
+    async def handle_facebook_poll(
+        self,
+        sender: p.Puppet,
+        thread_change: mqtt.ThreadChange,
```

Comment:
I guess backfilling polls isn't supported since this only handles MQTT? Not sure if we care though

Reply:
They are like properties of the thread in FB so that when you vote, the poll actually moves to the bottom of the thread.

I don't think we care about it.

Once we implement native bridging and votes and such, then we can handle the vote events and render the poll if it doesn't exist already.